src/harness/fourslash.ts

```diff
@@ -704,13 +704,55 @@ module FourSlash {
             }
         }
 
-        public verifyCompletionListDoesNotContain(symbol: string) {
+        /**
+         * Verfiy that the completion list does NOT contain the given symbol. If text, or documentation, or kind are provided,
+         * the list contains the symbol all given parameters must matched. When any parameter is omitted, the parameters is ignored during comparison.
+         */
+        public verifyCompletionListDoesNotContain(symbol: string, expectedText?: string, expectedDocumentation?: string, expectedKind?: string) {

```

Comment:
so for the completion list, I also want to make sure the following case
```typescript
class Foo {} -> additional parameter will be able to make sure that the list doesn't contain symbol with name Foo and of kind local class
var x = class Foo { /**/ } -> similar here but for kind class
```
Note: you can just pass the name if you don't kind the rest